src/index.ts

```diff
@@ -121,9 +119,41 @@ function handleCellClick(event: MouseEvent) {
   }
 }
 
+function performActions() {
+  for (let x = 0; x < fieldSizeX; x++) {
+    for (let y = 0; y < fieldSizeY; y++) {
+      const cell = field.getCell(x, y);
+      cell.action(x, y);
+    }
+  }
+  updateFieldImages();
+}
+
+function updateFieldImages() {
+  const cells = document.querySelectorAll('.cell');
```

Comment:
Get all cells in the begginning and store them in array?

Reply:
Ok